input/pagecontent/gaps-examples.md

```diff
@@ -39,32 +39,42 @@ The Figure 3-22 shows the workflow for gaps in care.
+This section contains an example that begins with a provider requesting a Gaps in Care Report for her patients from the payer. The provider receives the report, and she then submits additional data to the payer for the patient that was identified as having [open gaps] and [prospective gaps] in the report. She then later requests a Gaps in Care Report for these patients again from the payer.
 
 #### Step 1 - Initial Run for a Gaps in Care Report
-The resource graphs below represent the structure of the resources returned from the first [care-gaps](OperationDefinition-care-gaps.html) operation. Figure 3-23 shows the patient, *Gaps Patient01*, has an [open gap] because there were no resources in the payer system that would put her in numerator or denominator exclusion of the Colorectal Cancer Screening measure. The DetectedIssue resource contained in this Gaps in Care Report has the code "open-gap" as gap status indicating *Gaps Patient01* has an [open gap] for this measure. Figure 3-24 shows the second patient, *Gaps Patient02*, that is in the report has a [closed gap]. The `evaluatedResource` points to a colonoscopy procedure done in 2018 that had met the numerator criteria and resulted as a [closed gap]. Notice that the DetectedIssue resource's gap status code is "closed-gap", which indicates *Gaps Patient02* does not have an [open gap].
+The resource graphs below represent the structure of the resources returned from the first [care-gaps](OperationDefinition-care-gaps.html) operation.
+
+Figure 3-23 shows the patient, *Gaps Patient01*, has an [open gap] because there were no resources in the payer system that would put her in numerator or denominator exclusion of the Colorectal Cancer Screening measure. The DetectedIssue resource contained in this Gaps in Care Report has the code "open-gap" as gap status indicating *Gaps Patient01* has an [open gap] for this measure.
+
+Figure 3-24 shows the second patient, *Gaps Patient02*, that has a [closed gap]. The `evaluatedResource` points to a colonoscopy procedure done in 2018 that had met the numerator criteria and resulted as a [closed gap]. Notice that the DetectedIssue resource's gap status code is "closed-gap", which indicates that *Gaps Patient02* does not have an [open gap] or a [prospective gap].
+
+Figure 3-25 shows the third patient, *Gaps Patient03*, that has a [prospective gap]. The `evaluatedResource` points to a colonoscopy procedure done in 2011 that will be older than 10 years at the [gaps through period]. Because 10 years is the cutoff for a colonoscopy in the measure, the DetectedIssue resource's gap status code is "prospective-gap", which that a discrepancy will exist in the future between recommended best practices and the services that are actually provided and documented unless actions are taken.  
```

Comment:
this should actually say "2010" and not "2011"
```suggestion
Figure 3-25 shows the third patient, *Gaps Patient03*, that has a [prospective gap]. The `evaluatedResource` points to a colonoscopy procedure done in 2010 that will be older than 10 years at the [gaps through period]. Because 10 years is the cutoff for a colonoscopy in the measure, the DetectedIssue resource's gap status code is "prospective-gap", which that a discrepancy will exist in the future between recommended best practices and the services that are actually provided and documented unless actions are taken.  
```